src/components/TodoList/TodoList.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { Todo } from '../../types/Todo';
+import { TodoInfo } from '../TodoInfo/TodoInfo';
+
+type Props = {
+  todos: Todo[];
+  tempTodo: Todo | null;
+  onDelete: (todoId: number) => void;
+  loadingIds: number [];
+};
+
+export const TodoList: React.FC<Props> = ({
+  todos, tempTodo, onDelete, loadingIds,
+}) => {
+  return (
```

Comment:
Use the implicit return in arrow function for a cleaner code.

Reply:
done